azureiai/partner_center/plan.py

```diff
@@ -252,7 +251,7 @@ def create(self) -> dict:
         except ApiException as error:
             if args.update:
                 return self._update(args)
-            raise error
+            raise NameError("Plan already exists, try using update instead?") from error
```

Comment:
Is this the only exception that can come back? If not, this will mask the error that actually occurred in the case that the config is misconfigured

Reply:
if any other exceptions do come up, they are so unexpected i want them to pop all the way to the top. This is the only one I know of that I cut off to improve the output.

Reply:
Unlike Java... catching exceptions is an anti-pattern.